app/policies/tag_policy.rb

```diff
@@ -42,8 +42,11 @@ def permitted_attributes
       fields = %i[name slug description system_tag]
       fields << :type if @record.instance_of?(Tag)
       fields.push(partner_ids: [], user_ids: [])
-
-    elsif (user.tag_admin? && user.tags.include?(@record)) || user.partner_admin?
+    elsif user.tags.include?(@record)
+      fields = %i[name slug description]
+      fields << :type if @record.instance_of?(Tag)
```

Comment:
hmmm this is a good question, I replicated what the root user could do minus system tag. But looking at git blame for that line I get

```
ad12caf5 Ivan Kocienski (2023-02-16 15:23):
Tag Policy update to allow type to be set on create only
```

So maybe I shouldn't let a tag admin change the type of the tag.

...

Actually I definitely shouldn't because we only let people be tag admins of partnership tags!

Reply:
I have removed this ability from the tag admin

Reply:
good catch. i think i remember that we only allow the type field to be set if it is empty anyway